push_admitance_task:
  reference_frame: "base_link"
  target_position:
    x: 0.7
    y: 0.0
    z: 1.
  target_orientation:
    x: 0.0
    y: 0.707
    z: 0.0
    w: 0.707
